static/app/views/issueDetails/groupReplays/groupReplays.tsx

```diff
@@ -109,4 +197,17 @@ const StyledLayoutPage = styled(Layout.Page)`
   background-color: ${p => p.theme.background};
 `;
 
+const ReplayCountHeader = styled('div')`
+  display: flex;
+  align-items: center;
+  padding-bottom: ${p => 2 * p.theme.grid}px;
```

Comment:
I'd drop this padding, and add gap to `StyledLayoutPage` above.
Also drop the margin/padding thats on the loading placeholder.

that'll fix the problem where the player and table are touching:
<img width="468" alt="SCR-20240311-oeor" src="https://github.com/getsentry/sentry/assets/187460/abba49ce-76b3-4e3c-92a1-8a9567506673">